xapian-core/tests/api_weight.cc

```diff
@@ -766,6 +775,7 @@ class CheckStatsWeight : public Xapian::Weight {
 	TEST_REL(uniqueterms,>=,1);
 	TEST_REL(uniqueterms,<=,doclen);
 	TEST_REL(wdf,<=,wdf_upper);
+    	(void)wdfdocmax;
```

Comment:
The idea of this class is that it does sanity checks of the various statistics.  For `wdfdocmax` it's defined as the largest wdf in the document, so we can check that it's at least as large as the wdf we get passed.